modules/kubernetes/control-plane-logs/charts/vector/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: vector
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.3
```

Comment:
Why 0.1.3 ?

Reply:
Me performing changes and I need to bump then when working. I can set it to 0.1.0 or something before we merge.